hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteBlocksCommandHandler.java

```diff
@@ -282,6 +281,10 @@ private void processCmd(DeleteCmdInfo cmd) {
       List<Future> futures = new ArrayList<>();
       for (int i = 0; i < containerBlocks.size(); i++) {
         DeletedBlocksTransaction tx = containerBlocks.get(i);
+        // Container is being balanced, ignore this transaction
+        if (shouldAbandon(tx.getContainerID(), cmd.getContainer())) {
+          continue;
```

Comment:
Shouldn't we be throwing some kind of exception here to tell the user that the operation can't be performed right now (including a detailed reason message)? (the same notice is for DeleteContainerCommandHandler)

Reply:
@vtutrinov Thank you for the review. This PR has been separated to small PRs, could you help to review https://github.com/apache/ozone/pull/4887?

I will close this PR.